app/services/cloud_connector_service.rb

```diff
@@ -55,6 +55,7 @@ def send_to_cloud_controller
 
     VALID_STATUS_CODES.include?(response.code) ? @task.update!(:controller_message_id => JSON.parse(response.body)['id']) : task_failed(response.body)
   rescue => error
+    Rails.logger.error("Error sending message to connector: #{cc_url}")
```

Comment:
@hsong-rh This won't catch 404 errors since it will be processed by line 56 which calls task_failed()